addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -77,6 +78,7 @@ export const CROSS_TAB_CLIENT_MESSAGE = {
     UPDATE_VOLUME: "UPDATE_VOLUME", // sent by a tab to signal a volume change
 };
 const PING_INTERVAL = 30_000;
+const UNAVAILABLE_AS_REMOTE = "This action can only be done in the call tab.";
```

Comment:
This must have `_t()`. `_t(variable)` does not work.
`_t` 1st param is explicit string (not code evaluation, exact string in LOC) that is exported for translation.